skimage/segmentation/active_contour_model.py

```diff
@@ -48,6 +54,16 @@ def active_contour(image, snake, alpha=0.01, beta=0.1,
         Maximum iterations to optimize snake shape.
     convergence: float, optional
         Convergence criteria.
+    callback: function, optional
+        Called every 11 iterations with three args.

```

Comment:
Why not just call it each iteration?  The user can decide to skip over 10, if they choose to.